src/controllers/customercontroller.js

```diff
@@ -0,0 +1,70 @@
+class CustomerController extends Controller
+{
+    view()
+    {
+        this._customerHelper();
+    }
+
+    generateOrder()
+    {
+        var products = this.constructor._makeOrder();
```

Comment:
Hmm.. not working for me, since `_makeOrder` is a static function it gives an error in the console.